pkg/install/v1alpha1/jiva_pool.go

```diff
@@ -26,7 +26,7 @@ metadata:
   name: default
   type: hostdir
 spec:
-  path: "/var/openebs"
+  path: {{env "OPENEBS_IO_JIVA_POOL_DIR" | default "/var/openebs"}}
```

Comment:
let us put the sample yaml snippet i.e. maya api server deployment that makes use of this env & value

Reply:
In adition to this change, should `StoragePool` be ignored to be reapplied on an already installed setup?

Reply:
https://github.com/openebs/openebs/pull/2731